src/types/types.ts

```diff
@@ -1584,6 +1584,48 @@ export interface SendPrivateTransactionOptions {
   fast: boolean;
 }
 
+export enum AssetType {
```

Comment:
add jsdoc for every enum and property b/c that's what goes into the IDE autocompletion tip

Reply:
adding jsdoc for types in `types.ts` not the ones in `internal/raw-interfaces.ts`

Reply:
yeah sounds good!